.github/workflows/trigger-regression-tests.yml

```diff
@@ -0,0 +1,112 @@
+name: Trigger Regression Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Select the branch to run tests on'
+        required: true
+        default: 'main'
+  pull_request_target:
+    types: [labeled]
+
+env:
+  RUN_ID: ${{ github.run_id }}-${{ github.run_attempt }}
+
+jobs:
+  run:
+    if: >
+      (github.event_name == 'workflow_dispatch') ||
+      (github.event_name == 'pull_request_target' && github.event.label.name == 'regression-tests')
+    runs-on: ubuntu-latest
+    outputs:
+      branch_name: ${{ steps.set-branch.outputs.branch_name }}
+    steps:
+      - name: Set Branch Name
+        id: set-branch
+        run: |
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            echo "branch_name=${{ github.event.inputs.branch }}" >> $GITHUB_OUTPUT
+          else
+            echo "branch_name=${{ github.event.pull_request.base.ref }}" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.set-branch.outputs.branch_name }}
+
+  release:
+    needs: run
+    uses: bactions/workflows/.github/workflows/release-go-server.yml@main
+    with:
+      ref: ${{ needs.run.outputs.branch_name }}
+      version: ${{ github.run_id }}-${{ github.run_attempt }}
+      os: linux
+      cgo_enabled: true
+      release_binaries: false
+      docker_org: bsvb
+      docker_image_name: spv-wallet
+      release_create: false
+    secrets:
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
You need to add also:
```
DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
```
Otherwise the image won't be published, because workflow won't be able to log in to dockerhub.

Reply:
DOCKER_USERNAME added